src/complexity/languages-to-json-ast/java/services/init-generation-java.service.ts

```diff
@@ -0,0 +1,68 @@
+import * as fs from 'fs-extra';
```

Comment:
Afin d'éviter la duplication de code avec le fichier ts/init-generation.service.ts , il faudrait créer un répertoire languages-to-json-ast/core dans lequel placer tous les fichiers ou méthode ayant une utilité commune à tous les langages.